clippy_lints/src/stable_sort_primitive.rs

```diff
@@ -111,9 +112,10 @@ impl LateLintPass<'_> for StableSortPrimitive {
                 STABLE_SORT_PRIMITIVE,
                 expr.span,
                 format!(
-                    "used {} instead of {}",
+                    "used {} instead of {} to sort primitive type {}",
```

Comment:
NIT:

```suggestion
                    "used {} instead of {} to sort primitive type `{}`",
```

r=me with this change and reference files updated.